deploy/olm-catalog/common-service-operator/0.0.3/common-service-operator.v0.0.3.clusterserviceversion.yaml

```diff
@@ -182,6 +182,8 @@ spec:
         - apiGroups:
           - operator.ibm.com
           - operators.coreos.com
+          - etcd.database.coreos.com
```

Comment:
why add those two apigroups?

Reply:
These two apigroups are just for installing the example operators etcd and Jenkins and their cr.  These two apigroups will be removed after integrate with common services operators.